src/RadioButtons/index.scss

```
@@ -4,8 +4,8 @@
   .nds-label {
     display: block;
     position: relative;
-    padding-left: 30px;
-    margin-bottom: 12px;
+    padding-left: 34px;
+    margin-bottom: 16px;
```

```suggestion
    margin-bottom: var(--space-m);
```

good place for a spacing token

---

👍